test/scripts/python_tests.sh

```diff
@@ -0,0 +1,5 @@
+pip install -r cmd/suggestion/bayesianoptimization/requirements.txt
```

Comment:
You should add this script to https://github.com/kubeflow/katib/blob/master/test/scripts/unit-test.sh or add a new workflow for python test to argo.

Reply:
Thanks for catching this! I added it at the end of unit-test.sh